### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by ravikumar1002**

### CODE REVIEW

**Review by rohit8686 — APPROVED** [review]

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+}
+
+.btn-disabled-primary {
+  border: 1px solid var(--dark);
+}
+
+.btn-flote-primary {
+  box-shadow: -1px -1px 10px 5px rgb(161 177 188);
```
Comment (surajmore98) [inline components/button/button.css]: you can use variable for this color too.

**Review by surajmore98 — COMMENTED** [review]
overall looks good!

File: components/button/button.css (hunk 1 above)
Comment (ravikumar1002) [reply components/button/button.css]: I did, thanks for reviewing

**Review by surajmore98 — APPROVED** [review]

**Review by vinc3nati — APPROVED** [review]
Good to proceed.

**Pull request merged**